postgres-driver/load_balancer.go

```diff
@@ -73,3 +126,59 @@ func (d *PostgresDriver) ReadLoadBalancers() ([]*repository.LoadBalancer, error)
 
 	return loadbalancers, nil
 }
+
+// WriteLoadBalancer saves input load balancer in the database
+// Does not save stickiness configuration
+func (d *PostgresDriver) WriteLoadBalancer(loadBalancer *repository.LoadBalancer) error {
+	id, err := generateRandomID()
+	if err != nil {
+		return err
+	}
+
+	loadBalancer.ID = id
+	loadBalancer.CreatedAt = time.Now()
+	loadBalancer.UpdatedAt = time.Now()
+
+	insertLoadBalancer := extractInsertLoadBalancer(loadBalancer)
+	insertsLbApps := extractInsertLbApps(loadBalancer)
+
+	tx, err := d.Beginx()
+	if err != nil {
+		return err
+	}
+
+	_, err = tx.NamedExec(insertLoadBalancerScript, insertLoadBalancer)
+	if err != nil {
+		return err
+	}
+
+	for _, insert := range insertsLbApps {
+		_, err = tx.NamedExec(insertLbAppsScript, insert)
+		if err != nil {
+			return err
+		}
+	}
+
+	return tx.Commit()
+}
+
+// UpdateLoadBalancerOptions struct holding possible field to update
+type UpdateLoadBalancerOptions struct {
+	Name   string
+	UserID string
+}
+
+// UpdateLoadBalancer updates fields available in options in db
+func (d *PostgresDriver) UpdateLoadBalancer(id string, options *UpdateLoadBalancerOptions) error {
```

Comment:
We also need to be able to update the `UserID` and `UpdatedAt` fields, as part of the remove load balancer functionality.

Not sure if that should be a separate func?

Reply:
`UpdatedAt` is automatically updated with `time.Now` func, also added `UserID` field to application, it was already available in load balancer.

Reply:
Great.